conf/logrotate/cortx_ha_log.conf

```diff
@@ -4,7 +4,9 @@
     rotate 10
     size 10M
     compress
+    dateext
```

Comment:
https://github.com/Seagate/cortx-ha/pull/209 has 3 changed files while this one has two files ?

Reply:
Earlier PR related to log rotate was not ported to main branch. And just now got to know that Ajay P has already merged those changes yesterday to main branch. Need to revert those changes also.

Reply:
Rebased with latest main and resolved merge conflicts